owasp-zap/authenticator/.env.example

```
@@ -1,4 +1,20 @@
+
+# Provider Users pool (/v1/provider-users/*, /v1/purchases/*, GET attestation-by-id)
+PROVIDER_COGNITO_USER_POOL_ID=us-east-1_hijklmn
+PROVIDER_COGNITO_USER_POOL_CLIENT_ID=qrstuvwxyz0987654321
+PROVIDER_COGNITO_USERNAME=provider-zap@example.com
+PROVIDER_COGNITO_PASSWORD=456imapassword789
+
+# State Auth M2M pool (state-api.test.compactconnect.org)
+COGNITO_STATE_AUTH_DOMAIN=compact-connect-state-auth-test.auth.us-east-1.amazoncognito.com
+STATE_AUTH_CLIENT_ID=xxxxxxxxxxxxxxxxxxxxxxxxxx
+STATE_AUTH_CLIENT_SECRET=yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy
+STATE_AUTH_SCOPES=aslp/readGeneral aslp/write ky/aslp.write oh/aslp.write
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)